WordPress/Docs/WP/EnqueuedResourceParametersStandard.xml

```diff
@@ -0,0 +1,88 @@
+<documentation title="Enqueued Resource Parameters">
+    <standard>
+    <![CDATA[
+    The resource version must be set, to prevent the browser cache from retrieving and outdated version after updates.
```

Comment:
```suggestion
    The resource version must be set, to prevent the browser from retrieving an outdated, cached version after the resource has been updated.
```